src/GitHub.VisualStudio/UI/GitHubPane.cs

```diff
@@ -83,30 +80,38 @@ public GitHubPane() : base(null)
 
         protected override void Initialize()
         {
-            InitializeAsync().Catch(ShowError).Forget();
+            viewModelTask = ThreadHelper.JoinableTaskFactory.RunAsync(InitializeAsync);
```

Comment:
Can you have this class accept an `AsyncPackage` in construction and then use the `AsyncPackage.JoinableTaskFactory` instance instead of `ThreadHelper.JoinableTaskFactory` here?

That way if VS shuts down before this work is done, we won't risk crashing due to arbitrary work going on in background threads.